functions/exercise4/scriptSpect.js

```diff
@@ -0,0 +1,8 @@
+/// <reference path="../../typings/globals/jasmine/index.d.ts" />
+/*globals validatePhrase*/
```

Comment:
Please use more meaningful name that will briefly explain what are requirements against this phrase.